js/core/css/var.css

```diff
@@ -0,0 +1,18 @@
+.algorithm var:hover {
```

Comment:
`hover` won't work, as some algorithms are very long. For example: 
https://w3c.github.io/payment-request/#update-a-paymentrequest-s-details-algorithm

Ideally, want to be able to click, then scroll to see the instances.

Reply:
Argh, sorry... ignore this comment.